<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FOCUS-QA-4" name="com.focus.testcases.PurchaseOrdersTest" tests="18" failures="1" timestamp="10 May 2021 14:43:30 GMT" time="361.167" errors="0">
  <testcase name="checkLogoutAndLoginWithUserAllOptions" time="13.087" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkAuthorizationInPurchaseOrderVoucher2WithUserAllOptionsST" time="6.796" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkCloseRow2LinkInPurchaseOrderHavingTwoLinkDocuments" time="19.810" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkDataInPurchaseVoucherVATVoucher1ConvertedLink" time="18.962" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkLPurchaseOrderPopupInPurchaseVoucherVATVoucher2" time="20.459" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkDeletingVoucher2PurchaseOrder" time="15.197" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkPurchasesOrderVoucherToDisplayLinksWithoutAuthorization" time="32.835" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkLogoutAndLoginWithSUForPurchasesVoucherVAT" time="12.952" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkLogoutAndLoginWithUserAllOptionsSTToAuthorizePurchaseOrder" time="14.962" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkLogoutAndLoginWithSUAfterAuthorizingPurchaseOrderVoucher2" time="12.868" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkSavingPurchasesOrderVoucher2WithTwoRows" time="31.791" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkAuthorizationInMaterialRequisitionWithUserAllOptionsST" time="15.522" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkAuthorizationInPurchaseOrderWithUserAllOptionsST" time="15.450" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkVerifingDetailsOfSavedPurchaseOrdersVouchersInHomepage" time="14.845" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkPurchasesOrderVoucherToDisplayLinksAfterAuthorizationAndSavePurchaseOrder" time="27.935" classname="com.focus.testcases.PurchaseOrdersTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.PurchaseOrdersTest.checkPurchasesOrderVoucherToDisplayLinksAfterAuthorizationAndSavePurchaseOrder(PurchaseOrdersTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkPurchasesOrderVoucherToDisplayLinksAfterAuthorizationAndSavePurchaseOrder -->
  <testcase name="checkSavingPurchaseVoucherVATVoucher1WithPurchaseOrderLink" time="59.931" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkLogoutAndLoginWithUserAllOptionsSTToAuthorizePurchaseOrderVoucher2" time="14.908" classname="com.focus.testcases.PurchaseOrdersTest"/>
  <testcase name="checkLogoutAndLoginWithSU" time="12.857" classname="com.focus.testcases.PurchaseOrdersTest"/>
</testsuite> <!-- com.focus.testcases.PurchaseOrdersTest -->
